templates/worker-svc.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
```

Comment:
This seems identical to https://github.com/GoogleCloudPlatform/airflow-operator/blob/master/templates/svc.yaml
Do we need a new file ?

Reply:
The difference is there's a field `clusterIP: None` for creating the headless service.

Reply:
Ah i see. Then rename the file to headlesssvc.yaml and create tmplHeadlessSvc().

You do need 2 files.